### PR TITLE
fix(Box): Migrate backgroundColor to treat

### PR DESCRIPTION
**Pull request by michaeltaranto**

### CODE REVIEW

File: lib/hooks/useBox/box.treat.ts [hunk 1]
```diff
@@ -119,3 +119,7 @@ export const flexDirectionDesktop = css(({ utils: { desktopStyles } }) =>
     }),
   ),
 );
+
+export const backgroundColor = css(({ color: { background } }) =>
+  mapToCssRule(background, 'backgroundColor'),
+);
```
Comment (markdalgleish) [inline lib/hooks/useBox/box.treat.ts]: 👌

File: lib/components/Box/Box.tsx [hunk 2]
```diff
@@ -1,7 +1,11 @@
 import React from 'react';
-import useBox, { BoxProps } from '../../hooks/useBox';
+import useBox, { UseBoxProps } from '../../hooks/useBox';
 import { Reset } from '../Reset/Reset';
 import { ContrastProvider } from './ContrastContext';
+import { ResetProps } from '../../components/Reset/Reset';
```
Comment (mattsjones) [inline lib/components/Box/Box.tsx]: Imported this file twice

File: docs/src/App/Home/Home.tsx [hunk 3]
```diff
@@ -1,19 +1,22 @@
...
-  display: 'flex',
-  width: 'full',
-  marginLeft: ['none', 'xxsmall'],
-  marginRight: ['none', 'xxsmall'],
-  paddingBottom: ['xsmall', 'none'],
-};
+const Action = ({ children }: { children: ReactNode }) => (
+  <Box
+    display={'flex'}
+    width={'full'}
```
Comment (mattsjones) [inline docs/src/App/Home/Home.tsx]: `width="full"` ?

File: lib/themes/makeTreatTheme.ts [hunk 4]
```diff
@@ -45,6 +65,16 @@ const decorateTheme = (vars: ThemeVars) => ({
...
     },
+    background: {
+      ...vars.color.background,
+      formAccentActive: getActiveColor(vars.color.background.formAccent),
+      formAccentHover: getHoverColor(vars.color.background.formAccent),
+      brandAccentActive: getActiveColor(vars.color.background.brandAccent),
+      brandAccentHover: getHoverColor(vars.color.background.brandAccent),
+      infoLight: getLightVariant(vars.color.background.info),
+      criticalLight: getLightVariant(vars.color.background.critical),
+      positiveLight: getLightVariant(vars.color.background.positive),
```
Comment (mattsjones) [inline lib/themes/makeTreatTheme.ts]: I wonder if we should put these values on the theme or just the util itself and calculate where it's needed?

File: docs/src/App/Home/Home.tsx (hunk 3 above)
Comment (michaeltaranto) [reply docs/src/App/Home/Home.tsx]: Sorry, i was relying on prettier fixing my multi-cursor refactor laziness haha

Comment (mattsjones) [reply docs/src/App/Home/Home.tsx]: I assumed that'd be the case

File: lib/themes/makeTreatTheme.ts (hunk 4 above)
Comment (michaeltaranto) [reply lib/themes/makeTreatTheme.ts]: Hmm, i like the thinking, let me see what the implications are.

**Pull request merged**